config/schema/indexes/govuk.json

```diff
@@ -0,0 +1,5 @@
+{
+  "elasticsearch_types": [
+    "aaib_report"
```

Comment:
I think this could include all the document types at this stage, because

- We want to test we can populate the whole index asap, and measure the performance of that
- We're not planning to actually change the document types